packages/nimble-components/src/table/components/row/template.ts

```diff
@@ -69,7 +69,7 @@ export const template = html<TableRow>`
         `)}
 
         <span ${ref('cellContainer')} 
-            class="cell-container ${x => (x.isParentRow && x.nestingLevel > 0 ? 'nested-parent' : '')}"
+            class="cell-container ${x => (x.isNestedParent ? 'nested-parent' : '')}"
```

Comment:
oh this was an `&&` case.. maybe it's not clear to me why it needed to be marked volatile lol

Reply:
> Maybe bindings only track accessed observables the first time they are evaluated instead of every time? In which case yea we probably need to be much more thorough about correctly marking volatile access..

Ah, I guess that is exactly what volatile does:

> There is one special exception to this: if you have a computed property with branching code paths, such as ternary operators, if/else conditions, etc, then you must tell the observation system that your computed property has volatile dependencies. In other words, which properties need to be observed may change from invocation to invocation based on which code path executes.

https://www.fast.design/docs/fast-element/observables-and-state/

So actually we probably should go ahead and update the other bindings mentioned above that have short-circuits in their expressions to make sure they go through a volatile marked access.

We probably should make a tech debt to clean that up elsewhere.

Reply:
Created #1843

Reply:
fyi I think my follow-up comment confirmed that at least from docs, haven't checked source (may be comment racing)

Reply:
tech debt to handle looks good, resolving 👍